src/conftest.py

```diff
@@ -122,8 +125,6 @@ def evaluate(self, example: Example) -> None | str:
         import_header.append(
             path_to_import_statement(Path(os.path.relpath(example.path)))
         )
-        # Add microgrid as default import
-        import_header.append("from frequenz.sdk import microgrid")
```

Comment:
It is not clear to me why this import has been removed in `Fix some typing issues in src/conftest.py`

Reply:
Mmm, that's weird, I guess this was some problem with rebasing the PR because it wasn't intended. Good catch!

Reply:
Oh, I know. This is because I copied the "fixed" version from the channels repo, and the channels repo doesn't have the auto-import. I will remove the auto-import anyways (but in a different commit) to have the `conftest.py` unified between repos, as we should move it to `repo-config` and be general enough for all repos.